samples/event_manager/sample.yaml

```diff
@@ -1,12 +1,24 @@
 sample:
   description: Sample showing Event Manager usage
   name: Event Manager sample
+common:
+  tags: introduction
+  harness: console
+  integration_platforms:
+    - native_posix
+    - nrf52dk_nrf52832
+    - nrf52840dk_nrf52840
+    - nrf9160dk_nrf9160_ns
+    - nrf21540dk_nrf52840
+  harness_config:
+    type: multi_line
+    ordered: false
+    regex:
+      - ".*config_event.*"
+      - ".*measurement_event.*"
+      - ".*control_event.*"
+      - ".*ack_event.*"
+      - ".*Average value3: 45.*"
```

Comment:
Shouldn't it be:
`".*Average value3: 45"`
There are no trailing characters at the end of this line

Reply:
Actually we could remove `.*` from beginning and end of every regex and the test should work as expected.